SourceCode/Homework05/CRTRenderer.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include <iostream>
+#include <vector>
+#include <functional>
+#include "Utils/CRTTriangle.h"
+#include "Utils/CRTColor.h"
+#include "Utils/CRTScreen.h"
+
+using CRTScene = std::vector<CRTTriangle>;
+using CRTImage = std::vector<std::vector<CRTColor>>;
+
+class CRTRenderer
+{
+	CRTScreen screen;
+	CRTVector cameraPosition;
+public:
+	CRTRenderer(const CRTScreen& screen, const CRTVector& cameraPosition);
+
+	CRTImage renderScene(const CRTScene& scene, std::function<CRTColor(const CRTVector&, const CRTTriangle&, float)> visualization) const;

```

Comment:
The Scene is already a member in the class you could use it directly without passing it here.